### ISSUE TITLE
damage information on shears?

### ISSUE THREAD

**Issue by Minecraft2p0** [SquidDev-CC/CC-Tweaked#536]
I am very happy that I can use my sheep farm again thanks to the update. Unfortunately, the damage info is not integrated in turtle.getItemDetail (). Does it work any differently or is it yet to come?
So far it worked with data.damage ()

Translated with Google

**Comment by SquidDev** [comment]
`turtle.getItemDetail` changed slightly in 1.15 to make it a bit more thread-safe.

I've added a section to the [breaking changes](https://github.com/SquidDev-CC/CC-Tweaked/wiki/Incompatibilities-between-versions#minecraft-114-to-115) page, but basically you need to call `turtle.getItemDetail` with `true` as the second argument. This includes much more information (including damage), but takes a tick to execute.

**TLDR:** Replace `turtle.getItemDetail()` with `turtle.getItemDetail(nil, true)`.

**Comment by Minecraft2p0** [comment]
It works. Great. Thank you
